sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/CommunicationIdentityAsyncClient.java

```diff
@@ -304,17 +304,24 @@ public Mono<AccessToken> getTokenForTeamsUser(String teamsUserAadToken) {
         }
     }
 
+    private TeamsUserExchangeTokenRequest createTeamsUserExchangeTokenRequest(GetTokenForTeamsUserOptions options) {
```

Comment:
Q: Isn't it idiomatic in Java to sort members by the visibility modifiers? (like public first, private second...)

Reply:
I could not find any information regarding this in java docs or Azure guidelines, but I kind of used to sort them by calling order. Any idea on this @martinbarnas-ms?

Reply:
In the original Sun Java coding conventions the visibility order is suggested just for variables not for methods. Methods should be organized by functionality. I am not aware of any MS specific standard for it so I would keep methods related to each other close as @AikoBB did.

Reply:
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf

Reply:
ok, then I'd keep it below the `getTokenForTeamsUserWithResponse` right? since this one is using it too...

Reply:
done for both sync&async clients